index.html

```diff
@@ -64,6 +64,7 @@ <h2 class="break">Mission</h2>
                         style="color: #0000ee;"><span style="text-decoration: underline;">TTWG
                           Participants</span></span></a><br />
```

Comment:
I think the new link replaces the old link where we list TTWG participants, so should we remove that line as well?

Reply:
- changed participants link
- added agenda
- changed publications link